templates/page/page.tpl.php

```diff
@@ -0,0 +1,88 @@
+<?php
```

Comment:
I would like to split the template files up into a folder structure such as:

`html.tpl.php
block/
    block.tpl.php
    block--foo.tpl.php
field/
    field.tpl.php
    field--foo.tpl.php
page/
    page.tpl.php
    page-foo.tpl.php
views/
    views-field.tpl.php
    views-foo.tpl.php`

Reason being is that I've' experienced a lot of projects where people just stuff the templates directory full of, well, templates - which leads to a big messy list of files. This is just an effort to cleanup and organize the template files into logical folders to provide some structure and organization.